.gitignore

```diff
@@ -69,4 +69,5 @@ fastlane/screenshots/**/*.png
 fastlane/test_output
 
 # Random Mac files.
-.DS_Store
\ No newline at end of file
+.DS_Store
+Podfile.lock
```

Comment:
@bsiegel thoughts on this? I don't see any value in committing this file since everyone would pretty much need to run `pod install` locally anyhow

Reply:
Agreed, especially since we want to install pods fresh each time the CI runs this.